index.html

```diff
@@ -17,27 +17,39 @@
                 <a id="logo" href="#">Bengi Genç</a>
                 <ul class="desktop desktop-nav">
                     <li>
-                        <!-- link this to works section -->
                         <a class="desktop-links" href="#">Portfolio</a>
                     </li>
 
                     <li>
-                        <!-- link this to about page -->
                         <a class="desktop-links" href="#">About</a>
                     </li>
 
                     <li>
-                        <!-- link this to contact page -->
                         <a class="desktop-links" href="#">Contact</a>
                     </li>
-
                 </ul>
                 <button class="mobile" id="burger">
                     <span class="bar"></span>
                     <span class="bar"></span>
                     <span class="bar"></span>
                 </button>
             </nav>
+            <nav class="mobile-nav">
+                <ul class="mobile-nav-items">
+                    <li class="mobile-list-titles">
+                        <a class="mobile-links" href="#works">Portfolio</a>
+                    </li>
+
+                    <li class="mobile-list-titles">
+                        <a class="mobile-links" href="#about">About</a>
+                    </li>
+
+                    <li class="mobile-list-titles">
+                        <a class="mobile-links" href="#contact-page">Contact</a>
+                    </li>
+                </ul>
+
+            </nav>
```

Comment:
- [ ] It would be better for the background not to be scrollable when the mobile nav is open, please fix this 👍🏿